include/eosio/vm/x86_64.hpp

```diff
@@ -445,43 +445,64 @@ namespace eosio { namespace vm {
       }
 
       void emit_get_global(uint32_t globalidx) {
-         auto icount = variable_size_instr(13, 14);
+         auto icount = variable_size_instr(24, 42); // emit_setup_backtrace can be 0 or 9, and emit_restore_backtrace 0 or 9, the total of the rest 24
          auto& gl = _mod.globals[globalidx];
-         void *ptr = &gl.current.value;
```

Comment:
> Is there a reason for using `get_global_*` and `set_global_*` instead of just `_globals[globalidx].value` ?

At the time when `emit_get_global` and `emit_set_global` are called, `execution_conetxt` is not constructed yet; we don't have access to `_globals`. We also want to decouple the generated instructions from physical address of `execution_context`.